### DISCUSSION TITLE
Feature Request: Dropdown menu required for external links provided in Grafana dashboard.

### DISCUSSION THREAD

**Discussion by ErzaScarle** [grafana/grafana#63359] [Dashboards, Panels & Visualizations]
When we add links to other dashboards at the top of your current dashboard, it gives us with the option of "drop-down" style. Kindly incorporate similar UI for adding external links to the Grafana dashboards as well. 
Currently those links appear at the top right, if there are multiple links then it gets clumsy.
<img width="910" alt="Screenshot 2023-02-15 at 3 51 02 PM" src="https://user-images.githubusercontent.com/22257397/219001251-ae2711ff-336f-49b1-85aa-d97093950167.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82924. Feel free to continue the discussion around this there. Thank you!
